openjdk/excludes/ProblemList_openjdk8.txt

```diff
@@ -413,7 +413,7 @@ jdk/jfr/event/gc/stacktrace/TestG1OldAllocationPendingStackTrace.java https://gi
 jdk/jfr/event/gc/stacktrace/TestG1YoungAllocationPendingStackTrace.java https://github.com/adoptium/aqa-tests/issues/3301 linux-arm
 jdk/jfr/event/gc/stacktrace/TestMetaspaceG1GCAllocationPendingStackTrace.java https://github.com/adoptium/aqa-tests/issues/3301 linux-arm
 jdk/jfr/event/gc/stacktrace/TestParallelMarkSweepAllocationPendingStackTrace.java https://github.com/adoptium/aqa-tests/issues/3301 linux-arm
-jdk/jfr/event/oldobject/TestObjectDescription.java https://github.com/adoptium/aqa-tests/issues/3301 linux-arm
+jdk/jfr/event/oldobject/TestObjectDescription.java https://github.com/adoptium/aqa-tests/issues/3301,https://github.com/adoptium/aqa-tests/issues/4342 generic-all
```

Comment:
Typically, the issue linked in the Problemlist file would be the one that needs to be worked on to fix the problem for why we exclude the testcase in the first place (so, expect it to be an open issue that tracks a problem to be fixed).  We will eventually turn on automation that will query these issue links, and if the issues are closed, try to reinclude the testcases automatically.

Is the core issue related to https://bugs.openjdk.org/browse/JDK-8274560 or an as yet to be raised upstream bug?

Reply:
> 

I think it's different to https://bugs.openjdk.org/browse/JDK-8274560, JDK-8274560 fail cause use Shenandoah GC. 

jdk/jfr/event/oldobject/TestObjectDescription.java only fail on openjdk8 when the GC is default GC, and it seems be a testcase bug. When I add -Xmx128M jvm option to run this testcase, it will run pass.

I have creat another [issue](https://github.com/alibaba/dragonwell8/issues/473). I will try to creat a PR on jdk8u-dev repo later.

According to your advice, I think the issue link shoule be change to https://github.com/alibaba/dragonwell8/issues/473